packages/components/pie-radio-group/test/component/pie-radio-group.spec.ts

```diff
@@ -1,17 +1,151 @@
 import { test, expect } from '@sand4rt/experimental-ct-web';
-import { PieRadioGroup } from '../../src/index.ts';
+import { PieRadio } from '@justeattakeaway/pie-radio';
```

Comment:
If we're using `pie-radio` in the tests it might need to be added to `devDependencies`

Reply:
i wonder how it did work without being explicit? 
the same thing goes for the checkbox group 👀

Reply:
I think it _kind of_ works, but if you check the CI logs it actually tests the components before building it, which doesn't seem right to me. I wonder if adding the missing dependencies would fix this

Reply:
added it for both checkbox group and radio 🤝